internal/testing/cluster_test.go

```diff
@@ -198,6 +198,26 @@ func TestUploadDownload(t *testing.T) {
 			t.Fatal("unexpected")
 		}
 	}
+
+	// Check that the spending was recorded.
+	err = Retry(100, testBusFlushInterval, func() error {
+		contracts, err := cluster.Bus.ActiveContracts(context.Background())
+		if err != nil {
+			t.Fatal(err)
+		}
+		for _, c := range contracts {
```

Comment:
Is this the assertion we want? You are asserting we uploaded to and downloaded from every contract where I think we should check the contracts we know got UL/DL traffic.